src/test/fixtures/config/configuration-file.tsx

```diff
@@ -0,0 +1,81 @@
+import { join } from "path";
+import {
+  didCredential,
+  localCredential,
+  rinkebyCredential,
+  ropstenAWSCredential,
+  ropstenCredential,
+} from "./credential";
+import { documentFormsV2 } from "./forms/v2";
+import { documentFormsV3 } from "./forms/v3";
+import { documentFormsV2Local } from "./forms/local";
+import { generateConfigFile } from "./generator";
+import { EmptyConfig, Forms } from "./types";
+
+const configFile: EmptyConfig = {
+  network: "",
+  wallet: {},
+  forms: [],
+};
+
+const formsV2: Forms[] = [{ version: "v2", forms: documentFormsV2 }];
+const formsV3: Forms[] = [{ version: "v3", forms: documentFormsV3 }];
+const didFormsV2: Forms[] = [{ version: "v2-did", forms: documentFormsV2 }];
+const didFormsV3: Forms[] = [{ version: "v3-did", forms: documentFormsV3 }];
+const allForms: Forms[] = [
+  { version: "v2", forms: documentFormsV2 },
+  { version: "v3", forms: documentFormsV3 },
+];
+
+const V2_ROPSTEN_DIRECTORY = join(__dirname, "_generated-config-files", "v2", "sample-config-ropsten.json");
+const V2_AWS_ROPSTEN_DIRECTORY = join(__dirname, "_generated-config-files", "v2", "sample-config-ropsten-aws.json");
+const V2_RINKEBY_DIRECTORY = join(__dirname, "_generated-config-files", "v2", "sample-config-rinkeby.json");
+const V2_DID_DIRECTORY = join(__dirname, "_generated-config-files", "v2", "sample-config-did.json");
+
+const V3_ROPSTEN_DIRECTORY = join(__dirname, "_generated-config-files", "v3", "sample-config-ropsten.json");
+const V3_RINKEBY_DIRECTORY = join(__dirname, "_generated-config-files", "v3", "sample-config-rinkeby.json");
+const V3_DID_DIRECTORY = join(__dirname, "_generated-config-files", "v3", "sample-config-did.json");
+
+const ALL_FORMS_ROPSTEN_DIRECTORY = join(__dirname, "_generated-config-files", "sample-config-ropsten.json");
+
+generateConfigFile(configFile, formsV2, ropstenCredential, V2_ROPSTEN_DIRECTORY);
+generateConfigFile(configFile, formsV2, ropstenAWSCredential, V2_AWS_ROPSTEN_DIRECTORY);
```

Comment:
passing in as an object, so we no need care about params sequence wdyt? otherwise this is fine also

Reply:
good idea, changed to pass in as an object.